backend/lucene/src/main/java/org/hibernate/search/backend/lucene/document/model/impl/LuceneIndexModel.java

```diff
@@ -68,6 +68,15 @@ public ScopedAnalyzer getScopedAnalyzer() {
 		return scopedAnalyzer;
 	}
 
+	public String getNestedDocumentPath(String absoluteFieldPath) {
```

Comment:
Generally speaking, I would prefer always to delegate as much as possible to the server class, keeping the client call as simple as possible. So that the code could be spread among different classes. 

Following the principle of [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter).

But actually in this case we already expose the `LuceneIndexSchemaFieldNode` to the outside. So I'll put aside that principle :)

Reply:
Yes, it's not relevant in this case.